Automattic/wp-calypso
# Videopress onboarding v2 global style

#### Proposed Changes

This PR adds a global videopress.scss stylesheet to customize VideoPress onboarding styling. 

#### Testing Instructions
* Apply this PR
* Go to http://calypso.localhost:3000/setup/intro?flow=videopress
* ✅ The global style should be applied
* ✅ A "powered by VideoPress" logo should be visible at the bottom of the page
* ✅ When navigating, the step progress should be applied (yellow bar on top of the page)
